app/javascript/angular/code/controllers/_sessions_list_ctrl.js

```diff
@@ -171,6 +172,19 @@ export const SessionsListCtrl = (
       elmApp.ports.fetchSessions.subscribe(() => {
         sessions.fetch();
       });
+
+      elmApp.ports.highlightSessionMarker.subscribe(id => {
+        const session = sessionsUtils.find(sessions, id);
+
+        if (session === undefined) {
+          $scope.highlightedSessionMarker.setMap(null);
```

Comment:
What if we used a variable like `firstLoad` instead of the scope for the marker?